Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -400,6 +400,62 @@ DECLARE_SOA_TABLE(Run2V0s, "AOD", "RUN2V0", run2v0::BCId,
                   run2v0::BBFlag, run2v0::BGFlag);
 using Run2V0 = Run2V0s::iterator;
 
+// ---- MC tables ----
+
+namespace mcvtx
+{
+DECLARE_SOA_INDEX_COLUMN(BC, bc);
+DECLARE_SOA_COLUMN(GeneratorsID, generatorsID, short);
+DECLARE_SOA_COLUMN(X, x, float);
+DECLARE_SOA_COLUMN(Y, y, float);
+DECLARE_SOA_COLUMN(Z, z, float);
+DECLARE_SOA_COLUMN(T, t, float);
+DECLARE_SOA_COLUMN(Weight, weight, float);
+DECLARE_SOA_COLUMN(NProduced, nProduced, int);
+} // namespace mcvtx
+
+DECLARE_SOA_TABLE(McVtxs, "AOD", "MCVTX", o2::soa::Index<>, mcvtx::BCId,
+		  mcvtx::GeneratorsID,
+                  mcvtx::X, mcvtx::Y, mcvtx::Z, mcvtx::T, mcvtx::Weight,
+                  mcvtx::NProduced);
+using McVtx = McVtxs::iterator;
+
+namespace label
+{
+DECLARE_SOA_COLUMN(Label, label, int);
+DECLARE_SOA_COLUMN(LabelMask, labelMask, uint16_t);
+} // namespace label
```

Comment:
Could you add a comment explaining the bit meaning?
Here we can later add dynamic columns for the single bits

Reply:
I will copy it from the converter.

Reply:
It just came to my mind that @aalkin said it may be better to have bit columns instead of bit mask operations? @aalkin can you comment?

Reply:
Individual bit columns would be better, but before we have a syntax to "mass declare" a number of them with one line it is good enough to use bitmasks.

Reply:
How many will we have?

Reply:
16 bits